modules/ROOT/pages/export-app-data-to-csv.adoc

```diff
@@ -0,0 +1,34 @@
+= Export App Data to a CSV File
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In built-in and custom dashboards, you can export app data to a CSV file, which you can then use in other applications. The data export works on an individual chart level, so only data in the specified chart is exported. When you apply a filter to a dashboard, the exported data includes only the data included in the filter you set. For example, if you apply a time period filter, only the data for that time period is exported.   
+
+[NOTE]
+This feature requires a Titanium subscription.
+
+. In Anypoint Monitoring, click *Built-in dashboards* or *Custom dashboards* in the sidebar on the left. 
```

Comment:
bold for Anypoint Monitoring? In Anypoint Platform, click *Anypoint Monitoring*, then... for first task in a topic?